msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -337,6 +337,20 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to navigate to a message in individual and group chats in Teams:
+
+`http://teams.microsoft.com/l/message/{chatId}/{messageId}?context={contextObject}`
+
+Example: `http://dev.local/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context={"contextType"%3A"chat"}`
```

Comment:
teams.microsoft.com

Reply:
and confirm if the QP should be URL encoded.

Reply:
The url should be of this format
?context={"contextType":"chat"}
That whole thing percent encoded is 
?context=%7B%22contextType%22:%22chat%22%7D